### PR TITLE
Project Directory Setup, Navbar & Footer Implementation

### PR DESCRIPTION
**Pull request by kaifur-rahman**

### CODE REVIEW

File: frontend/src/components/navbar/navbarParent.jsx
Comment (ankitamk14) [inline frontend/src/components/navbar/navbarParent.jsx:1]: The Navbar component seems minimal, and if its reusability is not required, you could consider inlining it where it's used.
You may consider combining thses minimal components in a single file with generic name like Layout.jsx

File: frontend/public/heroSectionAsset.png
Comment (ankitamk14) [inline frontend/public/heroSectionAsset.png:1]: Is this image in the public domain, or does it have Creative Commons usage rights? Could you please provide the supporting certificate via email?

File: frontend/public/index.html [hunk 1]
```diff
@@ -11,7 +11,7 @@
     />
     <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
     <!--
-      manifest.json provides metadata used when your web app is installed on a
+      manifest.json provides metadta used when your web app is installed on a
       user's mobile device or desktop. See https://developers.google.com/web/fundamentals/web-app-manifest/
     -->
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
```
Comment (ankitamk14) [inline frontend/public/index.html:17]: Remove link tag referencing the manifest.json

File: frontend/public/robots.txt
Comment (ankitamk14) [inline frontend/public/robots.txt:1]: Please retain this file; do not remove it.

File: frontend/src/components/footer/footerParent.jsx
Comment (ankitamk14) [inline frontend/src/components/footer/footerParent.jsx:1]: The FooterParent component seems minimal, and if its reusability is not required, you could consider inlining it where it's used.

File: frontend/src/components/heroSection/heroParent.jsx
Comment (ankitamk14) [inline frontend/src/components/heroSection/heroParent.jsx:1]: The HeroSection component seems minimal, and if its reusability is not required, you could consider inlining it where it's used.

File: frontend/src/reportWebVitals.js
Comment (ankitamk14) [inline frontend/src/reportWebVitals.js:1]: Please retain this file; do not remove it.

**Review by ankitamk14 — CHANGES_REQUESTED** [review]
Please address the concerns raised in the comments.

**Comment by kaifur-rahman** [comment]
I have addressed all the changes and raised a new PR with more sections added in the front end.

**Pull request closed without merging**